content/en/apps/guides/hosting/4.x/backups.md

```diff
@@ -0,0 +1,123 @@
+---
+title: "Backups in CHT 4.x"
+linkTitle: "Backups"
+weight: 330
+description: >
+    Which data to backup when hosting the CHT 4.x
+---
+{{% alert title="Note" %}}
+This guide is about backups in CHT 4.x - there's the [self hosted guide for 3.x]({{< relref "apps/guides/hosting/3.x/self-hosting#backup" >}}) which includes backups for 3.x.
+{{% /alert %}}
+
+## Introduction 
+
+As CHT 4.x uses a container per service, the only data that needs to be backed up is:
+
+* CouchDB database
+* Docker Compose and `.env` files 
+* TLS certificates
+
+This is because Docker containers are inherently stateless so all the important binaries are already stored in [CHT's Docker images](https://gallery.ecr.aws/s5s3h4s7/).  Docker Compose files, including the `.env` file, store all of your deployment's configuration.  Finally, the TLS certificates should be backed up to ease recover speed.
```

Comment:
Slight grammatical improvement?

```suggestion
This is because Docker containers are inherently stateless so all the important binaries are already stored in [CHT's Docker images](https://gallery.ecr.aws/s5s3h4s7/).  Docker Compose files, including the `.env` file, store all of your deployment's configuration.  Finally, the TLS certificates should be backed up to reduce recovery time.
```